src/app/components/admin-form-create-event/admin-form-create-event.component.html

```diff
@@ -0,0 +1,72 @@
+
+<mat-tab-group>
+    <mat-tab label="Create Event" (click)="createEvent"> </mat-tab>
+    <mat-tab label="Manage Event" (click)="goToManage"> </mat-tab>
+  </mat-tab-group>
+
+  <p>Welcome, User!</p>
+ 
+
+  <mat-horizontal-stepper linear #stepper>
+    <mat-step>
+        <form [formGroup]="nameFormGroup">
+            <ng-template matStepLabel>Event Name</ng-template>
+            Event Name:
+            <mat-form-field>
+                <mat-label>
+                    <input matInput formControlName="eventName" required placeholder="Event Name...">
+                </mat-label>
+            </mat-form-field>
+            <div>
+            <button mat-raised-button matStepperNext color="primary">+ Event</button>
+            </div>
+        </form>
+    </mat-step>
+    <mat-step>
+        <ng-template matStepLabel>Timing</ng-template>
+        <div id="displaySlider"> Display Time (in seconds): 
+            <mat-slider 
+            [(ngModel)] = "value"
+            [thumbLabel] = "thumbLabel"
+            [min] = "displayMin"
+            [max] = "displayMax"
+            >   
+            </mat-slider>
+        </div>
+        <div> Frequency (in seconds): 
+            <mat-slider 
+            [(ngModel)] = "value2"   
+            [thumbLabel] = "thumbLabel"
+            [min] = "freqMin"
+            [max] = "freqMax"
+            >
+            </mat-slider>
+        </div>
+        <div>
+            <button mat-raised-button matStepperPrevious color="primary">Back</button>
+            <button mat-raised-button matStepperNext color="primary">Next</button>
+        </div>
+    </mat-step>
+      
+    <mat-step>
+        <ng-template matStepLabel>Content</ng-template>
+        <div style="border-style:solid">
+            Add Content
+            <input type="file">
+            <button mat-raised-button color="secondary">Upload</button>
+        </div>
+        <div style="border-style:solid">
+            Event Line Up
+            <img *ngFor="let image of images" src={{image.src}} style="height:100px;" cdkDropList>
+        </div>
+        <div style="border-style:solid">
+            Gallery
+            <!-- get images from database, display images here.
+             -->
```

Comment:
Good job implementing drag and drop.  there are still some needed refinements.  The pictures should be controlled by a drag-able space and snap into order.  I believe there are some examples on the angular material docs demonstrating such behavior